mod/assets/lib/card/assets.rb

```diff
@@ -30,6 +30,15 @@ def refresh_assets force: false
         generate_asset_output_files if force
       end
 
+      def wipe
+        Auth.as_bot do
+          %i[asset_input asset_output].each do |field|
+            Card.search(right: field).each(&:delete!)
+            Virtual.where(right_id: field.card_id).destroy_all
```

Comment:
Did this do the trick?

Reply:
It works, but this isn't what solved the problem. The issue was in the variables card and happened even when re-eating from a fresh deck.  Still, nice to have around.